diagnostic.md

```diff
@@ -92,7 +117,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Yellow text color
+sans-serif font family
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive.